Library/Formula/osquery.rb

```diff
@@ -44,6 +44,7 @@ def install
                                  "--record=installed.txt"}
     end
 
+    mkdir_p "#{buildpath}/generated"
```

Comment:
Sure but you could have CMake make the generated directory as the first thing it does, no?